cscs-checks/tools/jupyter/jupyter_tests.py

```diff
@@ -9,32 +9,36 @@
 
 @rfm.simple_test
 class JupyterHubSubmitTest(rfm.RunOnlyRegressionTest):
-    def __init__(self):
-        self.valid_systems = ['daint:jupyter_gpu', 'daint:jupyter_mc',
-                              'dom:jupyter_gpu', 'dom:jupyter_mc']
-        self.valid_prog_environs = ['*']
-        self.sourcesdir = None
-        self.executable = 'hostname'
-        self.time_limit = '1m'
-        self.max_pending_time = '7m'
+    valid_systems = ['daint:jupyter_gpu', 'daint:jupyter_mc',
+                     'dom:jupyter_gpu', 'dom:jupyter_mc']
+    valid_prog_environs = ['*']
```

Comment:
I guess you mean `builtin` here? Should we repeat the test for each programming environment?

Reply:
I took the liberty and changed it to `builtin`. Feel free to revert the changes if you think so.

Reply:
I do not understand the result of the `*` vs `builtin`... 
@vkarak has changed to `builtin`, but for me it shows as `*`...
Is there any reason why we have `*`?